template.yml

```diff
@@ -32,6 +32,7 @@ doc:
     required: true
   - id: subtitle
   - id: short_title
+  - id: short_citation
```

Comment:
Add this under 'options' and say it is a string and it should start to work!

Reply:
OK, I can confirm that that does the trick! New commit added. I'd be happy to document it, but it's not clear where myst-specific documentation goes.